packages/engine-server/src/markdown/remark/dendronPub.ts

```diff
@@ -116,8 +116,11 @@ class ImageNodeHandler extends DendronNodeHander {
       : cOpts?.assetsPrefix;
     const imageNode = node;
     if (assetsPrefix) {
-      imageNode.url =
-        "/" + _.trim(assetsPrefix, "/") + "/" + _.trim(imageNode.url, "/");
+      const imageUrl = _.trim(imageNode.url, "/");
+      // do not add assetPrefix for http/https url
+      imageNode.url = !imageUrl.match("(http(s?)://)")
```

Comment:
nitpick: there's a `isWebUri` helper function in `common-all` that could be used here